### PR TITLE
Git - unsafe repositories

### PR DESCRIPTION
**Pull request by lszomoru**

### CODE REVIEW

**Review by digitarald — APPROVED** [review]
LGTM, one thought I had was to mention the git CLI command which most other resources on this topic do; but we do link to the git config page already and the docs are about our commands.

File: docs/sourcecontrol/faq.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ One way to avoid these prompts is to set up a [credential helper](https://help.g
 
 Another option is to disable the auto fetch feature by changing the following setting: `"git.autofetch": false`.
 
+### Why is VS Code telling me that the git repository is potentially unsafe?
```
Comment (gregvanl) [inline docs/sourcecontrol/faq.md:68]: Did we want to go with:
"Why is VS Code warning that the git repository is potentially unsafe?"

"telling" is a little anthropomorphic and also not inclusive to deaf users

Comment (lszomoru) [reply docs/sourcecontrol/faq.md:68]: @gregvanl, no specific reason, and you are absolutely correct! 
Happy to rephrase if you have a better suggestion. Thanks!

Comment (gregvanl) [reply docs/sourcecontrol/faq.md:68]: Cool, I'll merge and update the wording when I do an edit pass on vnext.

**Pull request merged**